[PATCH 2/7] plat: mediatek: mt7988: remove unused bl2_dev_qspi.c

Remove unused bl2_dev_qspi.c from mt7988 platform

Change-Id: I198966baf9cd315cf10d1f9f61e528c9da3c5fe4
---
 plat/mediatek/mt7988/bl2/bl2_dev_qspi.c | 20 --------------------
 1 file changed, 20 deletions(-)
 delete mode 100644 plat/mediatek/mt7988/bl2/bl2_dev_qspi.c
diff --git a/plat/mediatek/mt7988/bl2/bl2_dev_qspi.c b/plat/mediatek/mt7988/bl2/bl2_dev_qspi.c
deleted file mode 100644
index 098f8b5c2..000000000

Change-Id: I198966baf9cd315cf10d1f9f61e528c9da3c5fe4
--- a/plat/mediatek/mt7988/bl2/bl2_dev_qspi.c
+++ /dev/null
@@ -1,20 +0,0 @@
-/*
- * Copyright (c) 2023, MediaTek Inc. All rights reserved.
- *
- * SPDX-License-Identifier: BSD-3-Clause
- */
-
-#include <boot_spi.h>
-
-#define MTK_QSPI_SRC_CLK		CLK_MPLL_D2
-
-uint32_t mtk_plat_get_qspi_src_clk(void)
-{
-	/* config GPIO pinmux to spi mode */
-	mtk_spi_gpio_init();
-
-	/* select 208M clk */
-	mtk_spi_source_clock_select(MTK_QSPI_SRC_CLK);
-
-	return MTK_QSPI_SRC_CLK;
-}
--
2.45.2

